lib/Exceptions.php

```diff
@@ -66,20 +66,8 @@ function buildGitHubIssueQuery($title, $body, $labels = null, $maintainer = null
 	return $uri;
 }
 
-/**
- * Returns the exception message as HTML string
- *
- * @param object $e Exception The exception to show
- * @param object $bridge object The bridge object
- * @return string|null Returns the exception as HTML string or null.
- *
- * @todo This function belongs inside a class
- */
-function buildBridgeException($e, $bridge){
-	if(( !($e instanceof \Exception) && !($e instanceof \Error)) || !($bridge instanceof \BridgeInterface)) {
-		return null;
-	}
-
+function buildBridgeException(\Throwable $e, $bridge)
```

Comment:
This removes the type of the bridge parameter and description of the return value, which I feel are not really self-evident. Maybe something like:


```suggestion
function convertBridgeExceptionToHtml(\Throwable $e, \BridgeInterface $bridge): string
```

Reply:
It's an internal function invoked twice. We don't have to typehint everything.

Reply:
Adding type hints has minimal cost, while they document in self-enforcing way how to call the function and what to expect as the return value. Moreover, current function calls are not the only only interactions we need to consider – IMO even more important is guiding developers trying to make sense of the code, either because they are new to the project, or because their built-in garbage collector ran since they last saw the code unit.

Reply:
I'll add typehints.